src/Symfony/Bundle/FrameworkBundle/Resources/config/cache_pools.xml

```diff
@@ -30,20 +30,32 @@
         </service>
 
         <service id="cache.adapter.apcu" class="Symfony\Component\Cache\Adapter\ApcuAdapter" abstract="true">
+            <tag name="monolog.logger" channel="cache" />
             <argument /> <!-- namespace -->
             <argument /> <!-- default lifetime -->
+            <call method="setLogger">
+                <argument type="service" id="logger" on-invalid="ignore" />
```

Comment:
`on-invalid="ignore"` is mandatory because the tag is not handled by the FrameworkBundle but by the MonologBundle, which is not a hard dep of it. This attribute (or its "null" variant) is used everywhere monlog.logger already is.